remodel/connection.py

```diff
@@ -1,5 +1,5 @@
 
-import rethinkdb as r
+import rethinkdb
```

Comment:
Strange, I had tests failing when I did this. Now it does not, must have been some dirt in the virtualenv or similar. Will definitely change back to r.